polkadot/runtime/parachains/src/builder.rs

```diff
@@ -624,7 +677,14 @@ impl<T: paras_inherent::Config> BenchBuilder<T> {
 					.cloned()
 					.unwrap_or(self.target_session);
 
-				let (para_id, core_idx, group_idx) = self.create_indexes(seed);
+				let para_id = ParaId::from(seed);
+				let core_idx = CoreIndex::from(current_core_idx);
+				current_core_idx +=1;
```

Comment:
the behaviour here is weird. `start..last` are now paraids. It'd be better if we had them represent cores and have this logic dispute specific cores (regardless of their paraids)

Reply:
There is more work to be done to support disputes on cores of elastic paras. Assuming elastic scaling not used, these are 1:1 mapped to cores. so it should be fine.

Reply:
yeah, I was suggesting have this PR include these changes if useful.

Anyway, I don't think this'd be useful for benchmarking. And for tests, I don't think I can make a lot of use of the BenchBuilder (besides for a test or two, which is still useful).

I already added a lot of tests for paras_inherent (for candidate sanitization mostly) which exercise corner cases that are not easily expressable in such a high-level interface.

So it's fine to keep as is